lib/Benchmark.rakumod

```diff
@@ -1,21 +1,50 @@
+my subset PIntD of Int where * > 0;
+
 my proto sub timethis(|) is export {*}
-my multi sub timethis(UInt $count, Str:D $code) {
+my multi sub timethis(PIntD $count, Str:D $code) {
     use MONKEY-SEE-NO-EVAL;
     timethis $count, { EVAL $code }
 }
 
-my multi sub timethis(UInt $count, &code) { 
+my multi sub timethis(PIntD $count, &code) { 
     my $start-time := time;
     code() for ^$count;
     my $end-time   := time;
     my $difference := $end-time - $start-time;
     ($start-time, $end-time, $difference, $difference / $count)
 }
 
-my sub timethese(UInt $count, %h) is export {
+my multi sub timethis(PIntD $count, &code, Bool :$statistics, --> Hash:D[Duration:D]) { 
```

Comment:
I think this should execute exactly when :statistics is provided as true. Right it is just an optional argument that doesn't really disambiguate the call, moreover it will execute for :!statistics as well, which is a bit weird.